src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -1127,4 +1127,7 @@ For more information, visit https://docs.nuget.org/docs/reference/command-line-r
   <data name="VerifyCommand_NotSupported" xml:space="preserve">
     <value>Package signature validation command is not supported on this platform.</value>
   </data>
+  <data name="Warning_MacCatalystXamarinIOSCompat" xml:space="preserve">
+    <value>'{0}' was resolved as a dependency of '{1}', but the dependency is using 'Xamarin.iOS' while '{1}' is using '{2}' as its TargetFramework. There might be compatibility issues when MacCatalyst projects depend on Xamarin.iOS projects.</value>
```

Comment:
Great warning message!

Reply:
thank you! It was really tricky to get right!! I'm glad this seems good haha.